apps/events/dashboard/tests.py

```diff
@@ -10,7 +12,7 @@
 
 
 def create_generic_attendance_event():
-    future = timezone.now() + timezone.timedelta(days=1)
+    future = datetime.datetime.now(tz=timezone.utc) + timezone.timedelta(days=1)
```

Comment:
Hmm, this may fix the problem here, but there seems to be some deeper thing behind this that is not fixed?

Reply:
Okay, I reverted it, since this didn't fix it. 
I genuinely have no clue why it gives a naive datetime, since it is aware when testing in my local shell (and DDF is by default timezone-aware), perhaps we need to override some Django-settings?